style.css

```diff
@@ -0,0 +1,204 @@
+* {
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none !important;
```

Comment:
Was it necessary to use !important here?

Reply:
not really necessary, didn't made any changes.